upi/openstack/down-load-balancers.yaml

```diff
@@ -43,14 +44,13 @@
   # for each service present on the cluster. Let's make
   # sure to remove the resources generated.
   - name: 'Remove the cluster load balancers'
-    os_loadbalancer:
-      name: "{{ item.name }}"
-      state: absent
-      wait: no
+    command:
+      cmd: "openstack loadbalancer delete --cascade {{ item.id }}"
     with_items: "{{ lbs_tagged.json.loadbalancers }}"
     when:
     - os_networking_type == "Kuryr"
     - versions | length > 0
+    - '"PENDING" not in item.provisioning_status'
```

Comment:
noob question: why not "pending"? What if it is an early teardown, and the LB didn't have the time to come up?

Reply:
@pierreprinetti Octavia does not allow to delete LB with provioning status PENDING-*, it would return an error. This way we avoid it.

Reply:
I added a note to the docs, for the user to retry the playbook once the lb is active.

Reply:
perfect, thanks!